template_src/www/index.html

```diff
@@ -31,8 +31,9 @@
         <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: gap: https://ssl.gstatic.com 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src *; img-src 'self' data: content:;">
         <meta name="format-detection" content="telephone=no">
         <meta name="msapplication-tap-highlight" content="no">
-        <meta name="viewport" content="initial-scale=1, width=device-width, viewport-fit=cover">
-        <link rel="stylesheet" type="text/css" href="css/index.css">
+        <meta charset="UTF-8">
```

Comment:
From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta):
> The `<meta>` element declaring the encoding must be inside the `<head>` element and **within the first 1024 bytes** of the HTML as some browsers only look at those bytes before choosing an encoding.

The condition in bold is actually violated thanks to our gigantic license header. To prevent this from happening, it's best practice to make the `<meta>` element declaring the encoding the first element in `<head>`.

Furthermore, I would prefer if you used the lowercase variant of `utf-8`.

Reply:
Thanks for reply,
Usually, lower case or upper case in this situation are same without any problems, but the standard definition from w3.org define the charset utf-8 in uppercase, and for what you mentioned, i will edit to be in the upper side of head section.

Regards

Reply:
You are right, the `charset` is not case-sensitive. It's just an aesthetic thing. When it is written in uppercase, I feel it sticks out from the rest of the mostly-lowercase HTML.

Reply:
You are right, making it small letter looks better, but when we coding for default projects, i think it could be better if we follow the standard coding style from default sources, and as w3.org, charset written by default with upper case letters.

Regards

Reply:
Actually in my experience lowercase is the de-facto standard in the industry. Examples from different starter templates:

- https://github.com/facebook/create-react-app/blob/0b4560094950dae2d1a34877475d62377ce78fb7/packages/cra-template/template/public/index.html#L4
- https://github.com/vuejs/vue-cli/blob/48413fa3511369f5356da450fb72b2d3f7511dc7/packages/%40vue/cli-service/lib/config/index-default.html#L4
- https://github.com/h5bp/html5-boilerplate/blob/e1c4afe46ab838e22236a3f475d1090c07f4de56/src/index.html#L5